src/components/Restoration/CorrectNameOptions.vue

```diff
@@ -205,7 +226,7 @@ export default class CorrectNameOptions extends Vue {
 }
 
 .action-btns {
-  margin: 30px 0;
+  margin-top: 30px;
```

Comment:
Example:
![image](https://user-images.githubusercontent.com/10002889/218227522-b12203a0-210c-4d19-8a71-0a444c9444ef.png)

Reply:
This component is now in BCRS shared repo.